pkg/provider/azure_loadbalancer.go

```diff
@@ -3267,13 +3267,29 @@ func (az *Cloud) ensurePIPTagged(service *v1.Service, pip *armnetwork.PublicIPAd
 
 // reconcilePublicIPs reconciles the PublicIP resources similar to how the LB is reconciled.
 func (az *Cloud) reconcilePublicIPs(ctx context.Context, clusterName string, service *v1.Service, lbName string, wantLb bool) ([]*armnetwork.PublicIPAddress, error) {
+	logger := klog.FromContext(ctx).WithName("reconcilePublicIPs").
+		WithValues("load-balancer", lbName)
```

Comment:
how about using the same log format for those keys (e.g. loadBalancer, resourceGroup (in the following lines)?

Reply:
done